arbitrum-docs/run-arbitrum-node/run-nitro-dev-node.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: 'How to run a local Nitro dev node'
+description: 'This page provides instructions for setting up and running a local Nitro dev node for contract testing and development.'
+author: mahsamoosavi
+sme: mahsamoosavi
+target_audience: 'Developers deploying smart contracts using Stylus'
+content_type: how-to
+sidebar_position: 5
+---
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.mdx';
+
+<PublicPreviewBannerPartial />
+
+## Overview
+
+This page provides step-by-step instructions for setting up and running a local Nitro node in `--dev` mode. This mode is ideal for developers who want to quickly test contracts using a single node, as it offers a simpler and faster setup compared to more complex environments.
+
+While some teams use `nitro-testnode` for testing cross-layer messaging, which involves launching both Geth as L1 and Nitro as L2, this setup can be more complex and time-consuming. If your primary goal is to test contracts on a local node without needing cross-layer interactions, Nitro's `--dev` mode offers a lightweight and efficient alternative.
+
+However, if you need more advanced functionality—such as cross-layer messaging, working with both L1 and L2 chains, or testing interactions between different layers—`nitro-testnode` is the preferred option. The testnode setup allows you to simulate a full L1-L2 environment, which is critical for those scenarios. See here for instructions.
+
+Note that Nitro `--dev` mode is particularly ideal for Stylus contract testing, as it is much lighter and faster to set up compared to the full nitro-testnode environment.
+
+## Prerequisites
+
+Before beginning, ensure [Docker](https://docs.docker.com/get-docker/) is installed and running on your machine.
+
+## Run the Nitro dev node
+
+Start the Nitro dev node in `--dev` mode using the following command. If the Docker image is not already present, Docker will automatically pull it for you.
+
+```bash
+docker run -it --rm --name nitro-dev -p 8547:8547 offchainlabs/nitro-node:v3.2.1-d81324d --dev --http.addr 0.0.0.0local Nitro dev node in `--dev` mode
```

Comment:
Looks like the "local Nitro dev node in `--dev` mode" at the end of this line should be removed